src/components/formik-input/FormikInput.tsx

```diff
@@ -43,21 +43,24 @@ const InputBoxRight = styled.div<{ error?: boolean; disabled?: boolean, focussed
   color: ${props => props.error ? `${styleVariables.red}` : `${styleVariables.grayLight}`};
   padding: 10px 10px 10px 10px;
   border-radius: 2.5px;
-  background-color: ${styleVariables.white};
+  background: ${styleVariables.colorWhite};
```

Comment:
Background-color gebruiken ipv. background wanneer je een achtergrondkleur wil instellen. Is explicieter - en daardoor minder bug gevoelig.